### PR TITLE
Docs/user creates project

### PR DESCRIPTION
**Pull request by suzalflueck**

### CODE REVIEW

File: app/docs/user-processes.md [hunk 1]
```diff
@@ -0,0 +1,486 @@
+# User Processes
+
+This document describes the processes or use cases that users engage in when interacting with our POC prototype application.
+
+### Login Process
```
Comment (SungHwan-Park) [inline app/docs/user-processes.md]: This is h3, did you mean it?

Comment (suzalflueck) [reply app/docs/user-processes.md]: Nope, thank you. Good catch!

File: app/docs/user-processes.md [hunk 2]
```diff
@@ -0,0 +1,486 @@
...
+
+> :warning: The login solution presented in this POC prototype is temporary and will be replaced with an existing authentication system such as KeyCloak.
+
+> **Who:** All User Personas
+> **When:** A registered user is not currently logged into the application.
```
Comment (SungHwan-Park) [inline app/docs/user-processes.md]: who and when are displayed in one line. it's annoying markdown requires additional empty line for a new line.

Comment (suzalflueck) [reply app/docs/user-processes.md]: Thanks, I'll fix it. It doesn't show in my preview plugin, so that's good to know.

**Review by SungHwan-Park — APPROVED** [review]

**Pull request merged**